preprocessing/src/main/java/ImageData.java

```diff
@@ -25,8 +25,8 @@
     transient private MatOfByte compressed_img;
 
     //Status of the representation
-    public enum ImageDataState {UNCOMPRESSED, COMPRESSED, ERROR}
-    private ImageDataState state;
+    //public enum ImageDataState {UNCOMPRESSED, COMPRESSED, ERROR}
+    private main.java.MetaImageData.ImageDataState state;
```

Comment:
Variable state was not visible int ImageData class. But, I removed it from ImageData class and changed the visibility of variable state in MetaImageData class to public